languages.json

```diff
@@ -1,5 +1,14 @@
 {
     "languages":{
+        "ABAP":{

```

Comment:
This should be `Abap`, for changing the user facing name you can add the `"name": "ABAP"` property.